src/tasks/computational.cairo

```diff
@@ -299,3 +303,28 @@ func extract_params_and_construct_task{
         ),
     );
 }
+
```

Comment:
@petscheit i think it should be defined in fetch_trait.cairo
this is more rustacian way

Reply:
otherwise all gucci

Reply:
I wanted to do that initially, but it results in circular imports :(